src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/ControllerAdvice.kt

```diff
@@ -175,7 +175,7 @@ class ControllerAdvice {
 
   @ExceptionHandler(ValidationException::class)
   fun handleValidationException(e: Exception): ResponseEntity<ErrorResponse> {
-    log.error("Validation exception {}", e)
+    log.error("Validation exception {}", e.message)
```

Comment:
don't need it for validation exception though, this was a mistake highlighted by intellij as there is a `{}` param